index.html

```diff
@@ -1,166 +0,0 @@
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <meta property="og:title" content="CV en ligne de David Cailleau"/>
-        <meta property="og:type" content="website"/>
-        <meta property="og:url" content="https://davidcailleau.github.io/CV_DC/"/>
-        <meta property="og:image" content="https://davidcailleau.github.io/CV_DC/image.jpg"/>
-        <meta property="og:description" content="CV en ligne de David Cailleau"/>
-        <link rel="Stylesheet" href="StyleCV.css"/>
-        <link rel="Stylesheet" media="screen" href="MediaCV.css"/>
-        <script src="https://kit.fontawesome.com/ffb09557c8.js" crossorigin="anonymous"></script>
-
-        <title>CV David Cailleau</title>
-
-    </head>
-
-    <body>
-        <div id="blocpage">
-            <div id="leftsection">
-                <figure id="photo_DavidC"><img src="images/DavidC.jpg" alt="Photo de David Cailleau" /></figure>
-                <section id="contact">
-                    <h3>contact</h3>
-                    <p>Adresse<br />
-                    Tigery, France</p>
-                    <p>Téléphone<br />
-                    06 62 06 XX XX</p>
-                    <p>Email<br />
-                    Cailleau.David@free.fr</p>
-                    <p>Site Web<br />
-                    www.name.com</p>
-                    <p>Skype<br />
-                    @nom_dutilisateur</p>
-                </section>
-
-                <section id="apropos">
-                    <h3>à propos de moi</h3>
-                    <p>Après 18 ans de carrière au sein de l'entreprise Accor, ma vie prend un nouveau tournant. 
-                        Je suis actuellement en congé de reclassement. Je mets celui-ci à profit pour me     reconvertir dans le 
-                        développement Web.
-                    </p>
-                </section>
-
-                <section id="loisirs">
-                    <h3>loisirs</h3>
-                    <div class="icon">
-                        <i class="fas fa-book fa-2x"></i>
-                        <i class="fas fa-film fa-2x"></i>
-                        <i class="fas fa-gamepad fa-2x"></i>
-                        <i class="fas fa-futbol fa-2x"></i>
-                        <i class="fas fa-music fa-2x"></i>
-                    </div>                            
-                </section>
-            </div>
-            <div id="rightsection">
-                <header>
-                    <h1><span>david</span> cailleau</h1>
-                    <h2><span>Dével</span>oppeur Web Freelance</h2>
-                </header>
-
-                <section>
-                    <section class="formation">
-                        <h3><i class="fas fa-university fa-2x"></i>formation</h3>
-
-                        <div class="formation1">
-                            <div class="date">
-                                <i class="far fa-dot-circle"></i>
```

Comment:
Jete un oeil au guide FontAwesome sur comment intégrer des icônes accessibles https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility

Ou bien une traduction ici: http://learn.chilidesign.fr/publication/utilisez-des-icones-de-maniere-accessible/

Reply:
Merci. J'ai modifié